SPECS/binutils/linker-script-readonly-keyword-support.patch

```diff
@@ -1,156 +0,0 @@
-From 6b86da53d5ee2022b9065f445d23356190380746 Mon Sep 17 00:00:00 2001
```

Comment:
This file should not be removed as it is used with binutils when bootstrapping the toolchain

Reply:
This patch is no longer needed in the latest version.